ccd-definition/CaseField.json

```diff
@@ -180,7 +180,7 @@
     "LiveFrom": "01/01/2017",
     "CaseTypeID": "UNSPECIFIED_CLAIMS",
     "ID": "claimStatementOfTruthLabel",
-    "Label": "The claimant believes that the facts in this claim form are true. \n\nI am duly authorised by the claimant to sign this statement.\n\nThe claimant understands that proceedings for contempt of court may be brought against anyone who makes, or causes to be made, a false statement in a document verified by a statement of truth without an honest belief in its truth.",
+    "Label": "The claimant believes that [the answers given] [the facts in the brief description of claim] are true. \n\nI am duly authorised by the claimant to sign this statement.\n\nThe claimant understands that proceedings for contempt of court may be brought against anyone who makes, or causes to be made, a false statement in a document verified by a statement of truth without an honest belief in its truth.",
```

Comment:
is this actually intended? it blows my mind what the judges are asking for

Reply:
yes did double chech with darren